pkg/util/profiling/profiling.go

```diff
@@ -48,6 +48,7 @@ func Start(settings Settings) error {
 		profiler.WithEnv(settings.Env),
 		profiler.WithService(settings.Service),
 		profiler.WithURL(settings.ProfilingURL),
+		profiler.WithDeltaProfiles(false),
```

Comment:
The docs for this fn state that

> This option takes precedence over the DD_PROFILING_DELTA

So I guess `DD_PROFILING_DELTA` would not enable this. Probably not an issue, only need to update commit/PR description.

Reply:
Oof thanks for having noticed this.

This is annoying because it means I'll have to add an Agent configuration field just for this then.